auraed/src/init/power.rs

```diff
@@ -88,10 +92,7 @@ pub(crate) fn spawn_thread_power_button_listener(
     std::thread::spawn(move || {
         loop {
             let event_slice = unsafe {
-                slice::from_raw_parts_mut(
-                    &mut event as *mut _ as *mut u8,
```

Comment:
I wonder if you could have removed one of the mut's here and have had it work?

Reply:
i tried a few different things with the cast, but only the coercion seemed to work.  there is an inline version with two `let` statements but it's uglier than the method, i think.